components/Footer.js

```diff
@@ -7,80 +7,124 @@ class Footer extends Component {
   render() {
     return (
       <StyledFooter>
-        <div className="newsletter-container">
-          <p className="newsletter-cta"> Get the Latest TJI Updates</p>
-          <p className="newsletter-subheading">Subscribe to our newsletter to get the latest updates:</p>
-          <MailChimpForm />
-        </div>
-
-        <div className="social-links">
-          <a
-            href="https://www.facebook.com/TXJusticeInitiative"
-            target="_blank"
-            title="Visit Texas Justice Initiative on Facebook"
-          >
-            <img src={require('../images/tji-fb-icon.svg')} alt="TJI Facebook" />
-          </a>
-          <a href="https://twitter.com/JusticeTexas" target="_blank" title="Visit Texas Justice Initiative on Twitter">
-            <img src={require('../images/tji-twitter-icon.svg')} alt="TJI Facebook" />
-          </a>
-        </div>
-
-        <div id="logo">
-          <Link href="/">
-            <img src={require('../images/tji-logo.svg')} alt="TJI Logo" />
-          </Link>
+        <div className="section-container">
+          <div className="section section__copyright">
+            <h2>Texas Justice Initiative</h2>
+            <p>
+              Copyright 2018 Texas Justice Initiative. All rights reserved.{' '}
+              <a href="http://texasjusticeinitiative.org/disclaimer">Disclaimer</a>.
+            </p>
+          </div>
+          <div className="section section__about-links">
+            <h4 class="section-title">About</h4>
+            <Link href="/about">About TJI</Link><br></br>
+            <Link href="/about-the-data">About the Data</Link>
+          </div>
+          <div className="section section__data-links">
+            <h4 class="section-title">Data</h4>
+            <Link href="/data">Explore the Data</Link><br></br>
+            <Link href="/publications">Publications</Link>
+          </div>
+          <div className="section section__get-involved-links">
+            <h4 class="section-title">Get Involved</h4>
+            <Link href="/contact">Volunteer</Link><br></br>
+            <Link href="/donate">Donate</Link><br></br>
+            <Link href="/contact">Contact Us</Link>
+          </div>
+          <div className="section section__social-links">
+            <h4 class="section-title">Follow Us</h4>
+            <a
```

Comment:
Jen, can you add rel="noopener" to the links that target new tabs? I left a comment on #46 with some details on why I think we should do this.